flags/src/main/java/com/yahoo/vespa/flags/Flags.java

```diff
@@ -196,6 +196,12 @@
             "Takes effect at redeployment",
             ZONE_ID, APPLICATION_ID);
 
+    public static final UnboundBooleanFlag USE_DIRECT_STORAGE_API_RPC = defineFeatureFlag(
+            "use-direct-storage-api-rpc", false,
+            "Whether to use direct RPC for Storage API communication between distributor and content node.",
```

Comment:
Could just leave this as "(...) communication between content cluster nodes.", as it also involves storage<->storage communication.

Reply:
Fixed